### PR TITLE
Shape inference for Conv2DTranspose

### PR DESCRIPTION
**Pull request by davideboschetto**
Wrong backend call on output shape computation in Conv2DTranspose

### Related Issues
Shape inference (implicit or explicit) was not working (undefined shapes). Now it works.

### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [X] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by davideboschetto** [comment]
Sorry for this wild tagging, but I'll take advantage of this and ask another question. Should we expect more active time for keras or should we just move to tf.keras, given that in the last few months there were no reviews and pull requests accepted?

Thanks, @gabrieldemarmiesse @Dref360 @ebursztein @farizrahman4u @haifeng-jin @taehoonlee

**Comment by fchollet** [comment]
There will be no further development of multi-backend Keras, and you should switch to tf.keras. Any PR should be directed to tf.keras.
